### ISSUE TITLE
failed to load or initialize dll. this probably means that a lut file is malformed or that dwm got updated.

### ISSUE THREAD

**Issue by DracuPlays** [lauralex/dwm_lut#52]
So I am on Win 11 OS Build 22621.3085, and for some reason i started getting this error "failed to load or initialize dll. this probably means that a lut file is malformed or that dwm got updated.", any fixes for that?

**Comment by lauralex** [comment]
@DracuPlays Did you download the 3.9.7 release? Also, did you try the previous version (3.9.6)?

**Comment by DracuPlays** [comment]
@lauralex yes, i tried almost all of the previous versions but unfortunately no luck, I was not having any issues before the Windows update, even tried uninstalling the latest Win 11 update after which I started getting the error.

**Comment by DracuPlays** [comment]
@lauralex I don't know how, but I got the idea of uninstalling and reinstalling "Microsoft Visual C++ Redistributables" and it fixed my issue somehow. I downloaded the "Visual Studio 2015, 2017, 2019, and 2022" from here (https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022) and reinstalled it on my system and boom, it worked. Anyway thanks alot for the updated versions od this program.
